monkey/monkey_island/cc/ui/src/components/report-components/AttackReport.js

```diff
@@ -157,11 +157,9 @@ class AttackReport extends React.Component {
         }
       }
     }
-    // modify techniques' messages
+    // compiles techniques' message string from markdown to HTML
     for (const tech_id in techniques){
-      if (typeof techniques[tech_id]['message'] === 'string') {
-        techniques[tech_id]['message'] = <div dangerouslySetInnerHTML={{__html: marked(techniques[tech_id]['message'])}} />;
-      }
+      techniques[tech_id]['message'] = <div dangerouslySetInnerHTML={{__html: marked(techniques[tech_id]['message_string'])}} />;
```

Comment:
Rather than change `message` -> `message_string`, what about storing the result of this line in `techniques[tech_id]['message_html']`? It would require other code changes, but the intent might be clearer.

We should also drop any unnecessary commits from the PR (force push) so we don't muddy up the commit history.

Reply:
Done